paper-ieee-trans.tex

```diff
@@ -66,12 +66,12 @@ \section{Introduction}
 supporting an unified access to large-scale computing and intense data analysis
 applications, has been driven by the needs of scientific communities as part of their
 pan-European research collaborations. One such example is the Worldwide
-Large Hadron Collider (LHC) Computing Grid infrastructure, motivated
+Large Hadron Collider (LHC) Computing Grid infrastructure, driven
 by the particle physics scientific community, built to satisfy the computing needs
 of processing, storing and analysis of the very large amount of data produced by the
 LHC particle accelerator. A significant number of innovation projects and
-technology transfer activities -- from the 5th, 6th and 7th Framework Programmes,
-and recently by the Horizon 2020 programme \cite{h2020} -- are under way to
+technology transfer activities -- from the European Union's funded 5th, 6th and 7th Framework Programmes,
+and recently from the Horizon 2020 programme \cite{h2020} -- have as their main goal to
```

Comment:
recently changed by mario, see https://github.com/orviz/paper-ieee-software-reliability/commit/0f03125846c685bfb6b5d6c374e5569e3f130e04#diff-82e2bed65915b5fe6d8aaf5df032545bR74

Reply:
not exactly rephrasing because the projects "are under way" is not exactly true. And it just came into my mind a rephrasing: instead of "provide a seamless and transparent access to distributed computing and data resources by the researchers." use "provide a seamless and transparent access of researchers to distributed computing and data resources."

Reply:
for me both are fine, don't have strong feelings

Reply:
agree with change
agree with Cris rephrasing

Reply:
I would use: 'provide researches with a seamless and transparent access to distributes computing and data resources.

I would use European Commission instead of European Union

Reply:
EC then